fontdrasil/Cargo.toml

```diff
@@ -12,6 +12,12 @@ categories = ["text-processing", "parsing", "graphics"]
 
 [dependencies]
 
+parking_lot = "0.12.1"
```

Comment:
where's parking_lot being used inside fontdrasil specifically? I see it imported in fontir and fontbe only.

Reply:
That's a leftover from my attempt to make identifiers unnecessarily complicated, removed, ty.